### ISSUE TITLE
Wireguard

### ISSUE THREAD

**Issue by Jenstel** [hufrea/byedpi#111]
Можно ли ваше решение использовать для обхода блокировки Wireguard, чтобы отправлять фейковые udp?

**Comment by hufrea** [comment]
Да, на Android это можно сделать в NekoBox. На десктопе, конечно, можно заморочиться с tun2socks/redsocks + wireproxy, но лучше использовать Zapret.
